spec/models/user_spec.rb

```diff
@@ -2,14 +2,34 @@
 
 RSpec.describe User, type: :model do
   describe "User#" do
-    let(:user) { FactoryGirl.create(:user) }
+    let(:user) { FactoryGirl.build(:user) }
+     
+    it "valid user" do
+      expect(user).to be_valid
+    end
 
     it "respond to email" do
       expect(user).to respond_to(:email)
     end
+    
+    describe 'email field' do
+      let(:user) { FactoryGirl.build(:user) }
+      let(:user2) { FactoryGirl.build(:user) }      
+ 
+      it 'email field empty' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.